src/fs_terminal_commands.h

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright (C) 2015 The University of Notre Dame
+This software is distributed under the GNU General Public License.
+See the file LICENSE for details.
+*/
+
+#ifndef FS_TERMINAL_COMMANDS_H
+#define FS_TERMINAL_COMMANDS_H
+
+#define ROOT "/"
+
+#define BUFFER_SIZE 256
+
+char cur_path[BUFFER_SIZE]; // The current working directory for the command line environment
```

Comment:
This should probably be in the .c, not the header? We don't want people to modify it themselves